internal/cluster/server/server_test.go

```diff
@@ -0,0 +1,143 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	protoc "github.com/pbinitiative/zenbpm/internal/cluster/command/proto"
+	"github.com/pbinitiative/zenbpm/internal/cluster/network"
+	"github.com/pbinitiative/zenbpm/internal/cluster/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func TestServer(t *testing.T) {
+	ctx := t.Context()
+
+	mux, err := network.NewMux("")
+	if err != nil {
```

Comment:
May I propose using the gocrest lib, which is already a dependency,
and this makes the test more compact/readable.
```go
then.AssertThat(t, err, is.Nil())
```